dart-common/src/main/java/com/f2prateek/dart/common/AbstractDartProcessor.java

```diff
@@ -99,27 +102,7 @@ public void setIsDebugEnabled(boolean isDebugEnabled) {
     this.isDebugEnabled = isDebugEnabled;
   }
 
-  protected Map<TypeElement, InjectionTarget> findAndParseTargets(RoundEnvironment env) {
```

Comment:
Both the InjectExtraProcessor and HensonProcessor were using this code. But indeed InjectExtraProcessor didn't need to parse @HensonNavigable Annotations.